examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,5658 @@
+{
```

Comment:
<p>Which workflow do you deploy here?</p><p> </p><p>It seems that you expected the categoried input to Triton?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>